keyboards/ergodox_ez/matrix.c

```diff
@@ -203,20 +207,30 @@ uint8_t matrix_scan(void)
 #endif
 
 #ifdef LEFT_LEDS
-     mcp23018_status = ergodox_left_leds_update();
+    mcp23018_status = ergodox_left_leds_update();
 #endif // LEFT_LEDS
-    for (uint8_t i = 0; i < MATRIX_ROWS; i++) {
+    for (uint8_t i = 0; i < 7; i++) {
         select_row(i);
-        wait_us(30);  // without this wait read unstable value.
+        // and select on left hand
+        select_row(i+7);
```

Comment:
@jackhumbert I am a little concerned of making changes so deep here. I can't say I totally understand this. Your thoughts?